searx/engines/__builtins__.pyi

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""The builtin types that are added to the global namespace of a module by the
+intended monkey patching of the engine modules.
+
+.. attention::
+
+   Monkey-patching modules is a practice from the past that shouldn't be
+   expanded upon.  In the long run, there should be an engine class that can be
+   inherited.  However, as long as this class doesn't exist, and as long as all
+   engine modules aren't converted to an engine class, these builtin types will
+   still be needed.
+"""
+from __future__ import annotations
+
+import logging
+
+logger: logging.Logger
+supported_languages: str
+language_aliases: str
+
+# from searx.engines.ENGINE_DEFAULT_ARGS
+about: dict[str, dict[str, str | None | bool]]
+categories: list[str]
+disabled: bool
+display_error_messages: bool
+enable_http: bool
```

Comment:
It's also possible to override values from https://github.com/searxng/searxng/blob/4942c9b914d45dae9aadf193c97ddeb4acc8a466/docs/admin/settings/settings_engines.rst#L60 here, e.g. `enable_http2`. I'm not sure if it makes sense to add them here?